api_shield_operations.go

```diff
@@ -0,0 +1,221 @@
+package cloudflare
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+	"time"
+)
+
+// APIShieldCreateOperation should be used when creating an operation in API Shield Endpoint Management.
+type APIShieldCreateOperation struct {
```

Comment:
check out the conventions for naming CRUD operation params - https://github.com/cloudflare/cloudflare-go/blob/master/docs/conventions.md#methods.

Reply:
fixed in https://github.com/cloudflare/cloudflare-go/pull/1397/commits/583a0d227aeb5d06bbfbb9d9df4aedf0eb887219

Reply:
note this struct is used as part of `CreateAPIShieldOperationsParams` - it's basically what the user is expected to set when creating an operation

I've update the name of this to `APIShieldBasicOperation` to (hopefully?) make this clearer